### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by xpt1x** [evandrocoan/AmxxEditor#10]
You can add support for getting syntax automatically just like other languages for sublime ( Ex : **HTML** )

Try pasting pawn code directly in new file .
**Result** : It wont take syntax ( of AmxmdxPawn ) automatically 

@evandrocoan You can add this functionality to support copy - pasting stuff 😈

**Comment by evandrocoan** [comment]
You can force all files to the amxx syntax by:
1. Opening a `.sma` file
1. And going on the menu:

   1. ![image](https://user-images.githubusercontent.com/5332158/42779939-38177834-8918-11e8-9887-b1e8cb8c2063.png)
   1. ![image](https://user-images.githubusercontent.com/5332158/42779881-12ed4a20-8918-11e8-91c0-2a184ced2448.png)

**Comment by xpt1x** [comment]
No, i mean it just takes the highlighting but you can't get Auto suggest features / AutoComplete working Unless you save it as **.sma** 

Only after saving the file it takes data from includes acc to plugin 😞 
Its not a problem just a suggestion if you can implement it in future maybe... 😄
